packages/core/src/controllers/pairing.ts

```diff
@@ -56,6 +57,7 @@ export class Pairing implements IPairing {
 
   private initialized = false;
   private storagePrefix = CORE_STORAGE_PREFIX;
+  private messageTypesToIgnore = [TYPE_1];
```

Comment:
To align naming here to not have `messageTypes` and `payloadTypes` concepts side by side:

```suggestion
  private ignoredPayloadTypes = [TYPE_1];
```